exporter/diagnostic_data_collector_test.go

```diff
@@ -108,6 +119,25 @@ func TestDiagnosticDataCollectorWithCompatibleMode(t *testing.T) {
 	assert.NoError(t, err)
 }
 
+func getVendor(t *testing.T, client *mongo.Client, ctx context.Context, logger *logrus.Logger) (string, error) {
+	buildInfoCmd := bson.D{bson.E{Key: "buildInfo", Value: 1}}
+	res := client.Database("admin").RunCommand(ctx, buildInfoCmd)
+
+	var buildInfoDoc bson.M
+	err := res.Decode(&buildInfoDoc)
+	if err != nil {
+		return "", errors.Wrap(err, "Failed to run buildInfo command")
+	}
+	_, ok := buildInfoDoc["psmdbVersion"]
+	if ok {
+		return "Percona Mongo", nil
```

Comment:
Let's use const for strings

Reply:
I guess, it doesn't make sense to use constants here for tests.

For source code I specified constants there. I don't like to reuse constants in tests from the source code, because otherwise if you accidentally change constant value then your tests will pass but they shouldn't.